templates/content-widget-job_listing.php

```diff
@@ -14,21 +14,29 @@
 if ( ! defined( 'ABSPATH' ) ) {
 	exit; // Exit if accessed directly
 }
+
 ?>
 <li <?php job_listing_class(); ?>>
 	<a href="<?php the_job_permalink(); ?>">
-		<div class="position">
-			<h3><?php wpjm_the_job_title(); ?></h3>
+		<?php if( $logo ) { ?>
```

Comment:
$logo feels a bit ambiguous here. Does $logo contain the logo?
I suspect renaming it $show_logo here: https://github.com/Automattic/WP-Job-Manager/pull/1509/files#diff-82ff8ab4c82f4f3218530ca410c88b81 might be clearer

Reply:
Fixed.